extension.json

```diff
@@ -144,6 +148,10 @@
 			"value": "",
 			"description": "If set, only allow users to request a SSL certificate on this wiki."
 		},
+		"RequestSSLDomainCheckCNAME": {
```

Comment:
Personally I think we should use a single associative array config, `RequestSSLDomainChecks` or something then an option for like `CNAME`, `A`, `AAAA`, and `NS` keys for supported options for checking.